### ISSUE TITLE
Hope to contact

### ISSUE THREAD

**Issue by KaiQK** [sski11santanu/cow-activity-classification-dl#1]
Hello, Mr. sikder, I'm very interested in your cow-activity-classification project. I hope I can learn from you. If it's convenient, please leave your email so that we can have in-depth communication in the future. Thank you! We look forward to your reply.

**Comment by sski11santanu** [comment]
I'd be really glad if I could help someone in my field of interest. I am a student currently and I spend my free time learning. I did this project as an internship assignment. If I had time I'd definitely look forward to this. But I can summarise this: 9-axis IMU data -> process it to make it compatible with a 2D CNN -> build the model -> split data into training and validation sets - > train the model (checkpoint the best model based on validation loss/accuracy, if you wish to) -> test it
